run.sh

```diff
@@ -0,0 +1,2 @@
+echo $XDG_DATA_HOME
+exec /app/extra/launcher/labymodlauncher
```

Comment:
Missing zypak here

Reply:
In what way? Is the environment variable `- --env=ZYPAK_SPAWN_LATEST_ON_REEXEC='1'` not sufficient?

Reply:
No you have to wrap it like `zypak-wrapper /app/extra/launcher/labymodlauncher`

https://github.com/refi64/zypak?tab=readme-ov-file#basic-usage

> Now, instead of running your Electron binary directly, call it via `zypak-wrapper PATH/TO/MY/ELECTRON/BINARY`

Reply:
Allright, so does that fit?

Reply:
should work now